index.html

```diff
@@ -0,0 +1,120 @@
+      href="https://fonts.googleapis.com/css2?family=Epilogue:ital,wght@0,500;0,700;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <script
+      src="https://kit.fontawesome.com/e53bf7a0b8.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Frontend Mentor | Intro section with dropdown navigation</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav-links">
+        <a href="#" class="logo">
+          <img src="assets/logo.svg" alt="Logo" />
+        </a>
+
+        <ul class="nav-items">
+          <li class="nav-item">
+            <a href="#features">Features</a>
+            <img src="assets/icon-arrow-down.svg" alt="Arrow Down" />
+          </li>
+          <li class="nav-item">
+            <a href="#company">Company</a>
+            <img src="assets/icon-arrow-down.svg" alt="Arrow Down" />
+          </li>
+          <li class="nav-item">
+            <a href="#careers">Careers</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="right-buttons">
+        <button type="button" class="login">Login</button>
+        <button type="button" class="register">Register</button>
+      </div>
+
+      <nav class="ham-menu">
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+
+      <div class="mobile-menu">
+        <nav class="nav-close">
+          <i class="fa-solid fa-times"></i>
+        </nav>
+        <ul class="menu-items">
+          <li class="menu-item">
+            <a href="#features">Features</a>
+            <img src="assets/icon-arrow-down.svg" alt="Arrow Down" />
+          </li>
+          <li class="menu-item">
+            <a href="#company">Company</a>
+            <img src="assets/icon-arrow-down.svg" alt="Arrow Down" />
+          </li>
+          <li class="menu-item">
+            <a href="#careers">Careers</a>
+          </li>
+          <li class="menu-item">
+            <a href="#about">About</a>
+          </li>
+          <div class="menu-buttons">
+            <button type="button" class="login">Login</button>
+            <button type="button" class="register">Register</button>
+          </div>
+        </ul>
+      </div>
+    </header>
+
+    <section class="main-section">
+      <div class="left">
+        <h1>Make remote work</h1>
+        <p>
+          Get your team in sync, no matter your location. Streamline processes,
+          create team rituals, and watch productivity soar.
+        </p>
+        <button>Learn More</button>
+      </div>
+      <div class="right">
+        <img src="assets/image-hero-desktop.png" alt="" />
+      </div>
+
+      <div class="sponsors">
+        <img src="assets/client-databiz.svg" alt="" />
+        <img src="assets/client-audiophile.svg" alt="" />
+        <img src="assets/client-meet.svg" alt="" />
+        <img src="assets/client-maker.svg" alt="" />
```

Comment:
You are almost there, just little corrections to be made. Pls provide **alt** messages for your images